# Summary for 11-16-2021 meeting

## In today's meeting, we discussed:

* The details can be found from the video recording on the [box](https://uchicago.app.box.com/folder/148831327443)

* We review the population AF files in details from the excel fiel

* We go through the model in the plink and how the p value is calculated between groups.

## the plan for this week:

You can follow the steps below for the tasks we discssed during our meeting. There are some hints and tips on the running the tasks.

### Step I Import the filterd files from last week

From the last week, you should have created a new binary dataset with only the filtered variants. 

* The following code can be used to calculate the cohort based MAF for the data.

```
plink --bfile XXXreplace with your file nameXXX --freq 'case-control'
```

This code can check if the file you generated last time is complete and workable for the additional analyses. If the file works, it will output the MAF files for the data. 

### Step II Calculate the p values between case-control for all SNPs.

You can use the following code to calculate the chisq and p value for the data:

```
plink --bfile XXXreplace with your file nameXXX --freq 'case-control' --assoc 'counts'
```

The output file will have format like 

```
XXXreplace with your file nameXXX.assoc 
```

The structure of the file will look like:

```
CHR SNP BP A1 C_A C_U A2 CHISQ P OR 
1 1_69849 69849 A 4 1 G 1.08 0.2987 3.049 
...
...
...
```
The column P is the unadjusted p values from the calculation. The details can be found on the [plink website](https://www.cog-genomics.org/plink/1.9/assoc) and [format guide](https://www.cog-genomics.org/plink/1.9/formats#assoc).


Tips:

You can try the following commands to see what the difference there:
```
plink --bfile XXXreplace with your file nameXXX --freq 'case-control' --assoc
```
or 

```
plink --bfile XXXreplace with your file nameXXX --freq 'case-control' --nonfounders --assoc 'counts'
```