# Summary for 12-07-2021 meeting

## In today's meeting, we discussed:

* The details can be found from the video recording on the [box](https://uchicago.app.box.com/folder/148831327443)

* We discuss the common ways to visualize the GWAS results.

* We go through a R code to visualize the results. 

## the plan for this week:

You can follow the steps below for the tasks we discssed during our meeting. There are some hints and tips on the running the tasks. You can find the Rmarkdown code we discussed today [here](https://ucdavis-bioinformatics-training.github.io/2021-July-Genome-Wide-Association-Studies/data_analysis/GWASvisualizations.html). There is another [good tutorial](https://www.r-graph-gallery.com/101_Manhattan_plot.html) to follow. 

### Step I Play with the toy data for visualization.

You will use gwasResults data as the toy data to play with visualization.

You will need to install the package qqman to use this toy data.

```
library(qqman)
```

and run

```
manhattan(gwasResults, chr="CHR", bp="BP", snp="SNP", p="P" )
```

### Step II Visulize the result using your plink output data.

You can use the results with .assoc and input it as a text table into R.

You will follow the common steps in the two R markdown files above to plot the manhattan plot as well as the CMplot.

### Step III Output the SNP with low p values.

You can select certain SNP that is low in pvalues and output them into a CSV file. Be sure to include the CHR, SNP, BP columns in the CSV file. We will use this file for annotation next week.